build-tools/xaprepare/xaprepare/ConfigAndData/Configurables.Windows.cs

```diff
@@ -5,8 +5,8 @@ namespace Xamarin.Android.Prepare
 {
 	partial class Configurables
 	{
-		const string CorrettoDistVersion = "8.222.10.3";
-		const string CorrettoUrlPathVersion = "8.222.10.1";
+		const string CorrettoDistVersion = "8.232.09.1";
+		const string CorrettoUrlPathVersion = CorrettoUrlPathVersion;
```

Comment:
```
Error CS0110: The evaluation of the constant value for 'Configurables.CorrettoUrlPathVersion' involves a circular definition
```

https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3286692&view=logs&jobId=bfe1c8f0-0562-5169-4501-8cf84831035d&j=035e6729-702a-53d4-bea9-761745a9f552&t=b83f01ea-e9c6-5934-65d6-623446bc17a6

I assume this should be:

```
const string CorrettoUrlPathVersion = CorrettoDistVersion;
```